api/src/main/scala/com/pennsieve/api/DataSetPublishingHelper.scala

```diff
@@ -926,6 +928,17 @@ case object DataSetPublishingHelper extends LazyLogging {
       newWorkflowEnabled <- secureContainer.organizationManager
         .hasFeatureFlagEnabled(organization.id, Feature.Publishing50Feature)
 
+      workflowId = defaultPublishingWorkflow match {
+        case "4" => Some(PublishingWorkflows.Version4)
+        case "5" => Some(PublishingWorkflows.Version5)
+        case "flag" =>
+          newWorkflowEnabled match {
+            case true => Some(PublishingWorkflows.Version5)
+            case false => Some(PublishingWorkflows.Version4)
+          }
+        case _ => Some(PublishingWorkflows.Version5)
+      }
+
```

Comment:
hi @muftring - I'm just trying to understand the different options. what is the significance of having a "default publishing workflow"? Do we not get the same outcome with just the feature flag? So by "default" its disabled for example and then enabled per org (or vice versa).

Minor: perhaps we can rename "flag" to something more descriptive: e.g "publishingVersion5FeatureFlagEnabled", or something a little more descriptive (or shorter :-)).

Reply:
Hi @edmore, 
Yes, the goal is to get all organizations using the Publishing 5.0 Workflow. The current default is to use the old workflow. The new workflow is enabled by feature flag, which for backend/API capabilities are enabled per organization. 

Adding the Publishing 5.0 Feature Flag during development last year permitted us to "pilot" the new workflow in a limited fashion, while also retaining the old workflow for every other organization. The objective now is to make sure all organizations use the new workflow in favor of the old, without having to enable it for every organization via feature flags -- we also do not want newly added organizations to unintentionally use the old workflow. 

We are still retaining the ability to use the old workflow for a period of time, which is why I implemented use of the config setting with a fallback to consulting the feature flag. The next effort (in the last quarter of this year) will be to completely deprecate and remove the old workflow code, thus preventing some tech debt from accumulating.

By "flag" here, are you referring to the values subject to the `match` expression? I was trying to keep these values short and simple.

Reply:
Ok, noted. 
Yes, the "flag" suggestion was just for readability sake.